tw-pnl.py

```diff
@@ -785,7 +785,7 @@ def check(all_wk, output_summary, output_csv, output_excel, tax_output, show, ve
         if tsubcode == 'Deposit':
             if description != 'ACH DEPOSIT':
                 tax_free = True
-        if tsubcode == 'Withdrawal' and not isnan(symbol):
+        if tsubcode == 'Withdrawal':
             tax_free = True
```

Comment:
```suggestion
        if tsubcode == 'Withdrawal':
            if isnan(symbol):
                tax_free = True
            else:
                tax_free = True
```
with that it's still working, but it doesn't make sense